src/archiefvernietigingscomponent/destruction/tasks.py

```diff
@@ -169,11 +170,19 @@ def complete_and_notify(list_id):
             assigned_archivaris = approval_review.author
             email = EmailMessage(
                 subject=report.title,
-                body=report.content.read().decode("utf8"),
+                body=_(
+                    "Destruction list '%(list)s' has been processed. "
+                    "The report of destruction is attached to this email."
+                )
+                % {"list": destruction_list.name,},
                 from_email=settings.DEFAULT_FROM_EMAIL,
                 to=[assigned_archivaris.email],
             )
-            email.content_subtype = "html"
+            email.attach(
+                filename=f"verklaring_van_vernietiging_{get_random_string(length=6)}.pdf",
```

Comment:
Why the random string?

Reply:
I thought that this way if the archivist receives multiple emails with different reports, they have different filenames.
Maybe it's not needed though

Reply:
I rather add the VL ID in the report filename in that case. The ID is something we can show in other places. A random string is not.

Reply:
I noticed that when the report is created, a filename is already created with `verklaring-van-vernietiging_` + the list name. So I changed the email filename to re-use the already existing filename.